merge_data.py

```diff
@@ -192,453 +194,65 @@ def tidy_categories(categories_string):
                 tidied_string = tidied_string[:-1]
         return tidied_string
 
-    ### Combining dataset categories
-    def combine_categories(dataset_row):
-        """Combine OriginalTags and ManualTags to get all tags
+    ### Creating dataset categories for ODS
+    def find_keyword(str_tofind, str_findin):
+        """Finds if single word or phrase exists in string
 
         Args:
-            dataset_row (dataframe): one row of the dataset set
-        """
-        combined_tags = []
-        if str(dataset_row["OriginalTags"]) != "nan":
-            combined_tags = combined_tags + str(dataset_row["OriginalTags"]).split(";")
-        if str(dataset_row["ManualTags"]) != "nan":
-            combined_tags = combined_tags + str(dataset_row["ManualTags"]).split(";")
-
-        combined_tags = ";".join(str(cat) for cat in set(combined_tags))
-        return combined_tags
+            str_tofind (str): the word or phrase to find
+            str_findin (str): the body of text to search in
 
-    data["OriginalTags"] = data["OriginalTags"].apply(tidy_categories)
-    data["ManualTags"] = data["ManualTags"].apply(tidy_categories)
-    data["CombinedTags"] = data.apply(lambda x: combine_categories(x), axis=1)
+        Returns:
+            boolean: True if match is found
+        """
+        if re.search(r"\b" + re.escape(str_tofind.lower()) + r"\b", str_findin.lower()):
```

Comment:
Micro optimisation, but instead of calling `lower` on all the strings, you could just use the [`re.I` flag](https://docs.python.org/3/library/re.html#re.I)